Dockerfile

```diff
@@ -20,7 +20,8 @@ RUN go mod vendor
 RUN go mod tidy
 ARG TEST="true"
 RUN if [ "$TEST" != "false" ]; then ./validate.sh ; fi
-RUN go build -mod=vendor -ldflags "-X github.com/prebid/prebid-server/version.Ver=`git describe --tags | sed 's/^v//'` -X github.com/prebid/prebid-server/version.Rev=`git rev-parse HEAD`" .
+RUN go install github.com/go-delve/delve/cmd/dlv@latest
```

Comment:
This line seems .. weird.

Reply:
It is weird. Debugging in Go is very different than debugging in Java. There is both a client and a server process to make it work.